index.html

```diff
@@ -12,6 +12,26 @@
 </head>
 
 <body>
+    
+  <div class="navigation">
+    
+        <img src="img/lambda-black.png" alt="strategy">
+
+
+        <nav>
+          <a href="#">Home</a>
+          <a href="http://127.0.0.1:5500/about.html#">About</a>
+          <a href="#">Products</a>
```

Comment:
Its cool to see you linking to the about page using a URL, this can also be done by just setting  `href="./about.html"` since it is in the same directory.